src/DynamoCore/Engine/NodeToCode/NodeToCode.cs

```diff
@@ -1210,6 +1210,26 @@ public static void ReplaceWithShortestQualifiedName(ClassTable classTable, IEnum
             }
             #endregion
 
+            #region Step 5 Escape StringInputNodes
```

Comment:
Theres a description of all the steps starting at line 1092, should we consider updating step 5 there?